charts/rancher-vsphere-csi/Chart.yaml

```diff
@@ -6,16 +6,16 @@ annotations:
   catalog.cattle.io/rancher-version: '>= 2.6.0-0 <= 2.6.99-0'
   catalog.cattle.io/release-name: vsphere-csi
 apiVersion: v1
-appVersion: 2.3.0
+appVersion: 2.4.1
```

Comment:
Can we make this `2.4.1-rancher1` to match the `version`?

Reply:
Updated the `appVersion` to match `version`